templates/scripts/setup-user-access.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
```

Comment:
How about calling this something like "setup-local-aws-config"?

Reply:
to "setup-local-config.sh"